frontend/src/app/main/ui/workspace/sidebar.cljs

```diff
@@ -251,11 +251,17 @@
          [:> comments-sidebar* {}]
 
          (true? is-history?)
-         [:> tab-switcher*
-          {:tabs #js [#js {:label (tr "workspace.versions.tab.history") :id "history" :content versions-tab}
-                      #js {:label (tr "workspace.versions.tab.actions") :id "actions" :content history-tab}]
-           :default-selected "history"
-           :class (stl/css :left-sidebar-tabs)}]
+         (let [tabs (mf/object
```

Comment:
`mf/object` is a new helper for creating js object at compile time; with proper props renaming to js camelCase convention